cmd/quickstart.go

```diff
@@ -124,7 +124,8 @@ func quickstartExec(cmd *cobra.Command, args []string) error {
 	}
 
 	if shouldCompile {
-		if err := run.Run(cmd.Context(), initialTarget, "", genVersion, "", "", "", false); err != nil {
+		step := run.NewWorkflowStep("ignored", nil)
```

Comment:
Would we consider visualizing from here as well. Looks like we aren't rendering the tea model

Reply:
Done 👍